Aplib.Tests/Desire/GoalTests.cs

```diff
@@ -0,0 +1,84 @@
+using Aplib.Core.Desire;
+using Aplib.Tests.Stubs.Desire;
+using FluentAssertions;
+
+namespace Aplib.Tests.Desire;
+
+public class GoalTests
+{
+    [Fact]
+    public void Goal_WhenConstructed_ContainsCorrectMetaData()
+    {
+        // Arrange
+        Tactic tactic = new TacticStub(() => { });
+        IGoalPredicate predicate = new ConstantGoalPredicate(0f);
+        var name = "Such a good goal name";
+        var description = "\"A lie is just a good story that someone ruined with the truth.\" - Barney Stinson";
+
+        // Act
+        Goal g = new(tactic, predicate, name, description); // Does not use helper method DefaultGoal on purpose
```

Comment:
No single-letter names as per Microsoft C# naming conventions.

Reply:
Agreed, but the editorconfig should warn me for this

Reply:
True